src/components/Tooltip/Tooltip.scss

```diff
@@ -34,7 +34,7 @@
   &__storycard {
     min-height: 500px;
     display: flex;
-    align-items: center;
+    align-items: center !important;
     justify-content: center;
```

Comment:
Can we move these styles to a different file (e.g. `Tooltip.stories.scss`) so we can differentiate between component styles and Storybook styles?